src/components/wizard/wizard.component.ts

```diff
@@ -146,19 +147,24 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
     }
 
     private _step: number = 0;
-    private _onDestroy = new Subject<void>();
-
-    ngAfterViewInit(): void {
+    protected _onDestroy = new Subject<void>();
+
+    constructor(protected _wizardService: WizardService<WizardStepComponent>) {
+        // watch for changes to valid subject
+        this._wizardService.valid$.pipe(
```

Comment:
We need to unsubscribe from this on component destroy. We can also remove the `{ }` from the subscribe function as there is only one statement.